src/tests/suite/daos_rebuild.c

```diff
@@ -144,6 +136,22 @@ rebuild_single_pool_target(test_arg_t *arg, d_rank_t failed_rank,
 	rebuild_targets(&arg, 1, &failed_rank, &failed_tgt, 1, false);
 }
 
+static void
+rebuild_add_back_tgts(test_arg_t **args, int args_nr, d_rank_t *failed_ranks,
+		      int *failed_tgts, int nr)
```

Comment:
can call rebuild_add_back_tgts() just within rebuild_targets()'s end?

Reply:
well, we might still do thing after rebuild_targets(), like io validation, probably sth else. So I would prefer to add back at the end of each sub tests, instead of rebuild_targets()? or I miss your point?

Reply:
if only for io validation seems can be done by register the rebuild_post_cb. But not a big deal.